src/SimpleAccounting/Properties/Resources.fr.resx

```diff
@@ -567,4 +567,22 @@ Sans une telle configuration pour au moins un compte, cette fonction est désact
   <data name="Word_Currency" xml:space="preserve">
     <value>Monnaie</value>
   </data>
+  <data name="AccountType_Asset" xml:space="preserve">
+    <value>Actif</value>
+  </data>
+  <data name="AccountType_Carryforward" xml:space="preserve">
+    <value>Reporter</value>
```

Comment:
I think that it should be "Report à nouveau" or "Report à nouveau (RAN)" instead of "Reporter". 
More details on https://fr.wikipedia.org/wiki/Report_%C3%A0_nouveau

Reply:
Thanks!